pkg/recorder/diskrecorder/diskrecorder.go

```diff
@@ -101,40 +101,93 @@ func (d *DiskRecorder) SaveAtPath(records record.MemoryRecords, path string) (re
 	return completed, nil
 }
 
+// Prune the archives when there are more than count archives
+func (d *DiskRecorder) PruneByCount(countTreshold int) error {
```

Comment:
It would be nice to have a test for this one :)

Reply:
Done